sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/engine/model/configuration/SprimberProperties.java

```diff
@@ -37,6 +37,10 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 
     private List<String> tagFilters = new ArrayList<>();
     private String featurePath;
+    private PoolExecutorProperties tcExecutorPool =
+            new PoolExecutorProperties(3, 3, 60, "TC-Executor-");
```

Comment:
These numbers  in the code look really confusing. What about moving them to properties?

Reply:
I would prefer to have that default properties file - a single entry point for all the default values.